docs/content/docs/updates/v1.0.0.md

```diff
@@ -6,12 +6,39 @@ summary: |
   The 1.0 release is the first major release of the Fluid Framework.
 ---
 
-## Patch Updates
+# 1.3.0
 
-### Update 1.0.1:
+## @fluidframework/fluid-static
+
+We are now surfacing errors on FluidContainer “disposed” event, so the application can appropriately react to faulty conditions, if container was closed due to an error.
+
+## Notable bug fixes
+
+- If a single AzureClient instance is used to open multiple documents/containers, the internal in-memory cache was not handling that scenario correctly, so the "latest" snapshots would be polluted by the first document loaded. The result of that would be container closing as "corrupted". This has been fixed.
```

Comment:
I left this content here on purpose for now, as we are thinking how can we leverage automation of commits content to generate initial release notes. The commit content was "If AzureClient is used to open multiple documents/containers, the internal in-memory cache was not handling that scenario correctly, so the "latest" snapshots would be polluted by the first document loaded.", which is correct but it is not on point around what was wrong and what works now from usage perspective.

Reply:
I like this in practice but imagine we will always need some kind of curation/smoothing around this content that we don't really expect from devs in commit notes.

Reply:
Agree, I think @tylerbutler called it out to be more of an initial draft. Looking back, most of the notes (if not all) would likely need curation/smoothing as you call it. So this would mostly be a list of commits that we can use as starting reference for what came in.